kolibri/plugins/learn/assets/src/state/actions/main.js

```diff
@@ -156,12 +156,18 @@ function updateContentNodeProgress(channelId, contentId, progressFraction) {
 }
 
 function setAndCheckChannels(store) {
-  return setChannelInfo(store).then(channels => {
-    if (!channels.length) {
-      router.replace({ name: PageNames.CONTENT_UNAVAILABLE });
+  return setChannelInfo(store).then(
+    () => {
+      const channels = getChannels(store.state);
```

Comment:
decided to use getter rather than add a return [here](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/state/actions.js#L369)